core/store/orm/config.go

```diff
@@ -221,6 +239,8 @@ func init() {
 	ChainSpecificDefaults[80001] = polygonMatic
 
 	ChainSpecificDefaults[100] = xDai
+
+	ChainSpecificDefaults[144545313136048] = arbitrum
```

Comment:
Is this mainnet?

Reply:
No it's not...I don't think arbitrum has a live mainnet yet.